RESOURCES.md

```diff
@@ -1,6 +1,10 @@
 # Developer Resources
 
-Below is a non-exhaustive list of developer resources related to Kentico Xperience and Kentico Kontent.
+Below is a non-exhaustive list of developer resources related to [Kentico Kontent](https://kontent.ai/), and [Kentico Xperience](https://xperience.io/) (formerly EMS).
+
+> Kentico EMS [was rebranded](https://xperience.io/discover/blog/2020-05/accelerate-your-digital-transformation) to Kentico Xperience in May, 2020. Any version of the product before version 13 is still known as Kentico EMS.
```

Comment:
I added this description here so we don't have to repeat "(formerly EMS)" everywhere below. We can remove it in the future when the naming is well understood.